### PR TITLE
feat: add random

### PR DESCRIPTION
**Pull request by Lampese**

### CODE REVIEW

File: random/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Moonbit/Core Random
+
+## Overview
+
+This is an efficient random number generation function based on the MT19947 algorithm, which supports generating random numbers of type Int, Int32, and Int64.
```
[the remark this answers is not in the document]
Comment (Lampese) [reply random/README.md:5]: Thanks! Your advice is very helpful!

**Comment by bobzhang** [comment]
hi @Lampese I made a separate branch and committed  changes (I can not push to your branch directly). Given your past contributions, you are welcome to join us as a collaborator, our email: jichuruanjian@idea.edu.cn

**Comment by fantix** [comment]
> I made a separate branch and committed changes

#171

**Comment by Lampese** [comment]
> hi @Lampese I made a separate branch and committed  changes (I can not push to your branch directly). Given your past contributions, you are welcome to join us as a collaborator, our email: jichuruanjian@idea.edu.cn

Email has been sent.

**Pull request closed without merging**